src/sass/base/_b-utils.scss

```diff
@@ -5,3 +5,43 @@
 .no-background-image {
   background-image: none;
 }
+
+.va-button-link {
```

Comment:
This is ugly, but we need a way to style buttons as links, since we're using the link element design for actions in places.